### PR TITLE
Updated gerard to discordjs 12

### PR DESCRIPTION
**Pull request by AntonyGarand**
PR where prettier was ran before and after

### CODE REVIEW

File: src/models/mongo.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ const models = {
                 ? ranking.brawlhalla_id
                 : ranking.brawlhalla_id_one + "+" + ranking.brawlhalla_id_two;
               Leaderboard.findOne({ bhid: bhid }, (err, entry) => {
-                if (err) u.alertError(err);
+                if (err) console.error(err);
```
Comment (AntonyGarand) [inline src/models/mongo.js:164]: To update, forgot to switch it back

File: src/base.js [hunk 2]
```diff
@@ -4,14 +4,16 @@ const Augur = require("augurbot"),
   path = require("path"),
   u = require("./utils/utils");
 
+const disabledCommands = ["twitch.js"];
```
Comment (AntonyGarand) [inline src/base.js:7]: I should test it with twitch enabled, didn't get an API key so I disabled it initially

File: src/base.js [hunk 3]
```diff
@@ -21,12 +23,12 @@ const Handler = new Augur.Handler(config, {
   clientOptions: {
     disableEveryone: true,
     ws: {
-      intents: 9474,
+      // intents: 9474
```
Comment (AntonyGarand) [inline src/base.js:26]: This initially caused crashes on my end, I'll re-enable it

File: src/commands/bhInfo.js [hunk 4]
```diff
@@ -73,8 +73,11 @@ async function feedEmbed(item) {
       .setDescription(
         (content ? `${content} ` : "") + `[[Read More]](${item.link[0]})`
       )
-      .setTimestamp(date)
-      .setURL(item.link[0]);
+      .setTimestamp(date);
+    if (item.link[0]) {
+      embed.setURL(item.link[0]);
+    }
```
Comment (AntonyGarand) [inline src/commands/bhInfo.js:79]: Caused an issue when there was no link, such as when I didn't set a URL for the home page

File: src/commands/help.js [hunk 5]
```diff
@@ -48,7 +50,7 @@ const Module = new Augur.Module().addCommand({
               true
             );
             if (i == 20) {
-              msg.author.send({ embed: embed });
+              msg.author.send({ embed: embed }).catch(console.error);
```
Comment (AntonyGarand) [inline src/commands/help.js:53]: Didn't use `u.alertError` as the error might be caused by the discord API, therefore I'd rather have it in the console

File: src/base.js (hunk 3 above)
Comment (Gaiwecoor) [reply src/base.js:26]: Fortunately, Discord has said they're not enforcing Intents just yet due to some miscommunications about it.

**Pull request merged**